src/components/TourRequest/TourRequest.jsx

```diff
@@ -284,7 +298,7 @@ export class TourRequest extends Component {
             />
           </Modal.Footer>
         </Modal>
-        <CardMatriculados numeroAlunos={this.state.nro_matriculados} />
+        <CardMatriculados numeroAlunos={this.state.escola.quantidade_alunos} />
```

Comment:
se nao tiver escola, qtd alunos nao vem tbm, dá erro.
this.state.escola.quantidade_alunos

- {if this.state.escola this.state.escola.quantidade_alunos }
- const {quantidade_alunios}  = this.state.escola (melhor)

Reply:
Foram tomadas as devidas modificações.